### ISSUE TITLE
Delay fps

### ISSUE THREAD

**Issue by nazri07** [alievk/avatarify#281]
How To Fix

C:\Users\User\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

**Comment by alievk** [comment]
It's just a warning
